azure-pipelines-build-devtest.yml

```diff
@@ -7,17 +7,20 @@ trigger:
     - develop
   paths:
     exclude:
-    - README.md
+    - '*release-please*'
+    - '*.md'
```

Comment:
Not sure if this needs to be `**/*.md` instead (because of `conf_parser/README.md`)